css/footer.scss

```diff
@@ -0,0 +1,28 @@
+@import "shared/shared";
+
+.footer {
+  margin-top: 50px;
+  background-color: $footerBgColor;
+
+  &__container {
+    padding: 50px 0;
+
+    &-copyright {
+      color: #949494;
+    }
+  }
+}
+
+.footer-social-links {
+  display: flex;
+
+  > *:not(:last-child) {
+    margin-right: 8px;
+  }
+
+  > * {
```

Comment:
* is not the best way to use css styles except cases when we can't avoid using it because browser reads css styles from right to left and such rule mean that browser will find all html tags first and when will try to compare parent tags with what you specified as parent element in css. To say shortly - it's not optimized way to write css.